src/benchq/resource_estimation/_footprint_analysis.py

```diff
@@ -54,82 +55,89 @@ def estimate_cost(self) -> CostEstimate:
                 factories as fast as the factories run.
             - The bottleneck time cost is waiting for magic states.
         """
-        logical_storage = int(
+        if self.t_count == 0 and self.toffoli_count == 0:
+            raise ValueError("Circuit must contain T gates and/or Toffoli gates.")
+
+        n_logical_qubits_used_for_computation = int(
             math.ceil(
                 self.max_allocated_logical_qubits
                 * (1 + self.routing_overhead_proportion)
             )
         )
-        storage_area = logical_storage * _physical_qubits_per_logical_qubit(
-            self.logical_data_qubit_distance
+        n_physical_qubits_used_for_computation = (
```

Comment:
`storage_area` seems better to me there's an additional overhead of qubits needed to just route information. They aren't really used for computation. There's probably a middle ground.

Reply:
I didn't like storage area because:
1. It obscures the fact that we're just counting qubits here.
2. It implies that these qubits are just being used a storage, not actively implementing a circuit. 
Maybe `n_physical_qubits_used_for_circuit_implementation`?

Reply:
Oh! Does `n_physical_qubits_used_for_clifford_circuit` work better? I think it's quite precise.

Reply:
I can go with that